### PR TITLE
Akamai controlbar

### PR DESCRIPTION
**Pull request by karimkozman**

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
- enabled / disabled button not working
- clear button not clearing the data (should start fresh like on reload) 
- change layout so its clear which graph is for which metric

File: src/components/dashjs-player/dashjs-player.css [hunk 1]
```diff
@@ -16,6 +16,324 @@ video {
...
   margin-left: auto;
   margin-right: auto;
-  display:block;
+  display: block;
   width: 100%;
-}
\ No newline at end of file
+}
+
+/**************************************************/
```
Comment (DanielHabenicht) [inline src/components/dashjs-player/dashjs-player.css:23]: If you like you can move the controlbar to another file, e.g. `controlbar.scss` and add an ` @import "controlbar.scss"` on top (you have to rename this file to `scss` aswell

Comment (karimkozman) [reply src/components/dashjs-player/dashjs-player.css:23]: Alright, will do

Comment (karimkozman) [reply src/components/dashjs-player/dashjs-player.css:23]: @DanielHabenicht I moved `controlbar.scss` and made ControlBar.js locally. Also added icons but still icons aren't showing for some reason even though the path is right.. would you happen to know why? p.s. I left controlbar styles in `dashjs-player.css ` for now as it gives ```
Unexpected token (Note that you need plugins to import files
           that are not JavaScript)
``` when I import it

Comment (DanielHabenicht) [reply src/components/dashjs-player/dashjs-player.css:23]: I will have a look in the afternoon

**Pull request merged**